File: problems/raidtime/attachments/template.cpp
#include <iostream>
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

int main()
{
    int w;
    cin >> w; cin.ignore();
    for (int i = 0; i < w; i++) {
        string wall;
        getline(cin, wall);
    }
    int d;
    cin >> d; cin.ignore();
    for (int i = 0; i < d; i++) {
        string door;
        getline(cin, door);
    }

    // Write an answer using cout. DON'T FORGET THE "<< endl"
    // To debug: cerr << "Debug messages..." << endl;

    cout << "0 C4 needed to go through 0 walls" << endl;
}

File: problems/raidtime/attachments/template.java
import java.util.*;
import java.io.*;
import java.math.*;

class Solution {

    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);
        int w = in.nextInt();
        if (in.hasNextLine()) {
            in.nextLine();
        }
        for (int i = 0; i < w; i++) {
            String wall = in.nextLine();
        }
        int d = in.nextInt();
        if (in.hasNextLine()) {
            in.nextLine();
        }
        for (int i = 0; i < d; i++) {
            String door = in.nextLine();
        }

        // Write an answer using System.out.println()
        // To debug: System.err.println("Debug messages...");

        System.out.println("0 C4 needed to go through 0 walls");
    }
}
